src/Symfony/Bridge/Twig/composer.json

```diff
@@ -27,7 +27,7 @@
         "symfony/finder": "^4.4|^5.0",
         "symfony/form": "^5.0",
         "symfony/http-foundation": "^4.4|^5.0",
-        "symfony/http-kernel": "^4.4|^5.0",
+        "symfony/http-kernel": "^5.0",
```

Comment:
I think all similar changes should be reverted - DataCollectorInterface v4.4 is compatible with the new signatue.
Same for all components.